test/testfixture/recipe_funcs.go

```diff
@@ -573,6 +573,37 @@ func Trackers(n int, fns ...CustomizeTrackerFunc) RecipeFunction {
 	}
 }
 
+// CustomizeTrackerQueriesFunc is directly compatible with CustomizeEntityFunc
+// but it can only be used for the Trackers() recipe-function.
```

Comment:
Please change to `// but it can only be used for the TrackersQueries() recipe-function.`